doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -178,7 +182,7 @@ The Cloud Infrastructure must also provide the mechanism to identify corrupted d
 
 #### 6.3.3.1 Confidentiality and Integrity of communications
 
-It is essential to secure the infrastructure from external attacks. To counter this threat, API endpoints exposed to external networks must be protected by either a rate-limiting proxy or web application firewall and must be placed behind a reverse HTTPS proxy. Attacks can also be generated by corrupted internal components, and for this reason, it is security best practice to ensure integrity and confidentiality of all network communications (internal and external) by using Transport Layer Security (TLS) protocol.
+It is essential to secure the infrastructure from external attacks. To counter this threat, API endpoints exposed to external networks must be protected by either a rate-limiting proxy or web application firewall and must be placed behind a reverse HTTPS proxy (sec.mon.008). Attacks can also be generated by corrupted internal components, and for this reason, it is security best practice to ensure integrity and confidentiality of all network communications (internal and external) by using Transport Layer Security (TLS) protocol.
 When using TLS, according to the [OpenStack security guide](https://docs.openstack.org/security-guide/secure-communication/introduction-to-ssl-and-tls.html) recommendation, the minimum version to be used is TLS 1.2.
 
 3 categories of traffic will be protected using TLS:
```

Comment:
We don't specify what these 3 categories are...? 

Also we need to take care... external API endpoints should absolutely be protected as https but I recall that if you enable https on the internal endpoints you have a management headache with regards to the certificate management - without a suitable automation / deployment tool this can lead to operational problems as certificates start to expire.

Reply:
@iangardner22 the three categories are in the bullets that follow -- are you asking for changes to these bullets?

Reply:
Oh that wasn't visible in the rich-text - apologies.  All good